github/repos_deployments.go

```diff
@@ -137,7 +137,7 @@ type DeploymentStatus struct {
 	Description   *string    `json:"description,omitempty"`
 	TargetURL     *string    `json:"target_url,omitempty"`
 	CreatedAt     *Timestamp `json:"created_at,omitempty"`
-	UpdatedAt     *Timestamp `json:"pushed_at,omitempty"`
+	PushedAt      *Timestamp `json:"pushed_at,omitempty"`
```

Comment:
Now that I've looked, are we sure `pushed_at` is the right JSON field?

It doesn't look to be. I think it might be `updated_at` instead.

Take a look at https://developer.github.com/v3/repos/deployments/. There are 7 hits for "updated_at", 0 for "pushed_at".